packages/integration/src/mocks/auth/get_settings.ts

```diff
@@ -33,7 +33,7 @@ export default function mock_get_settings(context: Context) {
                 last_name: 'Smith',
                 non_pep_declaration: 1,
                 phone: '+66111111111',
-                place_of_birth: null,
+                place_of_birth: 'Thailand',
```

Comment:
🌴

Reply:
Have to include this if not It's gonna break integration test 😎